tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/dialog/extension-install-dialog.js

```diff
@@ -0,0 +1,307 @@
+
+
+                    //extension array from backend which has details about extensions.
+                    var extensionLists = (!_.isNil(this.extensionList) ? this.extensionList : getExtensionDetail());
+                    var extensionTable = $('<table class="table table-hover data-table"' +
+                        ' id="extensionTableId"><tbody></tbody></table>');
+                    //define the map to store Partially extension modal based on key
+                    var partialExtensionDetailModal = new Map();
+                    extensionLists.forEach(function (extension) {
+                        var extensionTableBodyData;
+                        if (extension.status.trim().toLowerCase() === constants.EXTENSION_INSTALLED) {
+                            extensionTableBodyData = $('<tr><td>' + extension.name + '</td><td>Installed</td><td><button' +
+                                ' class="btn btn-block btn' +
+                                ' btn-primary">UnInstall</button></td></tr>');
+                            extensionTableBodyData.find("button").click(function () {
+                                extensionUpdate(extension);
+                            });
+                        } else if (extension.status.trim().toLowerCase() === constants.EXTENSION_PARTIALLY_INSTALLED) {
+                            var partialExtensionIndex = extensionLists.indexOf(extension);
+                            extensionTableBodyData = $('<tr><td>' + extension.name + '</td><td>Partially-Installed' +
+                                '&nbsp; &nbsp;<a data-toggle="modal"' +
+                                ' id="' + partialExtensionIndex + '"><i class="fw' +
+                                ' fw-info"></i></a></td><td><button' +
+                                ' class="btn btn-block btn' +
+                                ' btn-primary">UnInstall</button></td></tr>');
+
+                            var partialModel = $(
+                                '<div class="modal fade" id="' + partialExtensionIndex + '">' +
+                                '<div class="modal-dialog">' +
+                                '<div class="modal-content">' +
+                                '<div class="modal-header"> ' +
+                                "<button type='button' class='close' data-dismiss='modal' aria-label='Close'>" +
+                                "<i class=\"fw fw-cancel  about-dialog-close\"></i>" +
+                                "</button>" +
+                                '<h3 class="modal-title file-dialog-title" id="partialExtenName">'
+                                + extension.name +
+                                '</h3> ' +
+                                '<hr class="style1"> ' +
+                                '</div>' +
+                                ' <div class="modal-body">' +
+                                ' <h3>Description</h3>' +
```

Comment:
Try to add a long description and see how UI behaves ,, whether they aligned correctly if not fix them using css

Reply:
9ed0740 this commit will be resolving above request.
![InfoModel](https://user-images.githubusercontent.com/40277159/72871139-1cca1c00-3d10-11ea-8d73-99054b4d1c7a.png)